manifests/install/archive.pp

```diff
@@ -12,6 +12,10 @@
   if !defined(Class['archive']) {
     include archive
   }
+  
+  Archive {
+    provider => 'curl',
```

Comment:
I think this should be incorporated to the `archive` resource block at line 46, since it's the only resource of type `archive` in this class.

Otherwise 🆗